pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.openmrs.owa</groupId>
+    <artifactId>orderentry</artifactId>
+    <version>1.2.4-SNAPSHOT</version>
```

Comment:
@ibacher - I agree, and that plugin is a cool find, I'd definitely be up for using it.  I was also worried about this at first, but then realized that:
a) We were already maintaining this version twice between package.json and bintray.json
b) The deploy itself will fail if these don't match, as the deploy step is looking for a file named orderentry-<maven-version>.jar, which will only succeed if these are aligned.

So I think b) covers us in the sense that it will be obvious when running a build or trying to deploy that these don't match and the build will fail.